profiler/profiler_quickstart/busybench.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START profiler_quickstart]
+
+// Sample profiler_quickstart simulates a CPU-intensive workload for profiler.
+package main
+
+import (
+	"log"
+	"runtime"
+
+	"cloud.google.com/go/profiler"
+)
+
+func busyloop() {
+	for {
+		load()
+		runtime.Gosched()
+	}
+}
+
+func load() {
+	for i := 0; i < (1 << 20); i++ {
+	}
+}
+
+func main() {
+	runtime.GOMAXPROCS(8)
```

Comment:
Why 8? Default seems fine to me.

If you disagree, please add a comment to the sample explaining why.